python/src/scipp/plotting/model.py

```diff
@@ -104,19 +107,34 @@ def _axis_coord_and_formatter(self, dim, data_array, dim_to_shape,
 
         contains_strings = False
         contains_vectors = False
+        contains_datetime = False
+        offset = 0.0
+        coord_label = None
+        form = None
 
         has_no_coord = dim not in data_array.meta
         if not has_no_coord:
             if data_array.meta[dim].dtype == sc.dtype.vector_3_float64:
                 contains_vectors = True
             elif data_array.meta[dim].dtype == sc.dtype.string:
                 contains_strings = True
+            elif data_array.meta[dim].dtype == sc.dtype.datetime64:
+                contains_datetime = True
 
         # Get the coordinate from the DataArray or generate a fake one
         if has_no_coord or contains_vectors or contains_strings:
             coord = make_fake_coord(dim, dim_to_shape[dim] + 1)
             if not has_no_coord:
                 coord.unit = data_array.meta[dim].unit
+        elif contains_datetime:
+            nanoseconds = data_array.meta[dim].values.astype(
+                'datetime64[ns]').astype(np.int64)
+            offset = np.amin(nanoseconds)
+            nanoseconds -= offset
+            coord = sc.Variable(dims=data_array.meta[dim].dims,
+                                unit=data_array.meta[dim].unit,
+                                values=nanoseconds,
+                                dtype=sc.dtype.int64)
```

Comment:
What is wrong with using
```suggestion
            coords = coord - sc.min(coord)
```

Reply:
1. the conversion to integers
2. i need the offset further down
3. `sc.min` doesn't work on datetime

Reply:
- I thought you want to convert to integers?
- I added `sc.min` in #1779.
- `offset = sc.min(coord)`, maybe converted to int if required?